src/main/content/_assets/js/header.js

```diff
@@ -28,4 +26,40 @@ $(document).ready(function() {
         }
     });
 
+    if ($('#whatsNewModal').attr('href')) {
+        $("#modal-body").load($('#whatsNewModal').attr('href'), function(response, status) {
+            if (status === "success") {
+                let version = `${$('#general_title').text().replace(/[^\d.]/g, '')}`;
+                let localStorageValue = localStorage.getItem('whatsNew');
+                localStorageValue = localStorageValue ? JSON.parse(localStorageValue) : {};
+        
+                if (version === localStorageValue['whatsNewVersion']) {
+                    if (!localStorageValue['didOpenWhatsNew']) {
+                        $('.toast').toast('show');
+                    }
+                }
+                else if (version !== localStorageValue['whatsNewVersion']) {
```

Comment:
no need for an else if, just an `else` would suffice since its the opposite condition of the if

Reply:
We would need an else if statment because the first condition essentially checks if a user has already opened the notification/modal...the second statment checks the current version  of the site against what's in local storage thus being two differnt entities we check for.